src/main/resources/services/rss/rss.js

```diff
@@ -0,0 +1,37 @@
+const portalLib = require('/lib/xp/portal');
+const contentLib = require('/lib/xp/content');
+const rssXmlLib = require('/lib/rss-xml');
+
+exports.get = function(req) {
+    const params = req.params;
+
+    if (params.method === 'list') {
```

Comment:
I would have used header "Content-Type" to determine if we should response with the xml or json data. 
But this should also do the trick 👍 Nice service